src/components/overlay/overlay.tsx

```diff
@@ -85,7 +85,7 @@ class Overlay extends Component<OverlayProps, any> {
     } else {
       // Remove the overlay-active class only when overlay portal has a single child
       const overlayPortalEl = getOverlayPortalElement(this.props.player);
-      if (overlayPortalEl!.childElementCount <= 1) {
+      if (overlayPortalEl && overlayPortalEl.childElementCount <= 1) {
```

Comment:
with TS you can use `?`
`overlayPortalEl?.childElementCount <= 1`

Reply:
No, because if either `overlayPortalEl` or `childElementCount` are `undefined`, you then the expression will be resolved as `undefined <= 1`, which isn't allowed in TypeScript.